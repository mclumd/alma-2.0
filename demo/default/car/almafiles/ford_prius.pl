% Example 1: we see a car, and we see it has an empty gas tank
obs(quote(rel(gas_tank_car, ford1))).
obs(quote(not(rel(fueled, ford1)))).

% Example 2: a charged prius with an empty gas tank
obs(quote(rel(electric_car, prius1))).
obs(quote(rel(gas_tank_car, prius1))).
obs(quote(rel(charged, prius1))).
obs(quote(not(rel(fueled, prius1)))).
