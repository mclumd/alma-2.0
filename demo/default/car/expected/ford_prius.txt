rel(car, ford1)
rel(gas_tank_car, ford1)
~rel(powered, ford1)
~rel(fueled, ford1)
abnormal(ford1, car, "~rel(powered, ford1)")
rel(car, prius1)
rel(gas_tank_car, prius1)
rel(electric_car, prius1)
rel(charged, prius1)
~rel(fueled, prius1)
rel(powered, prius1)
abnormal(prius1, gas_tank_car, "rel(powered, prius1)")
